libraries/AP_RCTelemetry/AP_RCTelemetry.cpp

```diff
@@ -84,7 +84,7 @@ void AP_RCTelemetry::update_avg_packet_rate()
 /*
  * WFQ scheduler
  */
-void AP_RCTelemetry::run_wfq_scheduler(void)
+uint8_t AP_RCTelemetry::run_wfq_scheduler(const bool use_shaper)
```

Comment:
Please comment on what the return value of this function is.

Reply:
ok done